pkgs/stdenv/generic/make-derivation.nix

```diff
@@ -252,6 +253,41 @@ in rec {
           ++ lib.optional (stdenv.buildPlatform.uname.system != null) "-DCMAKE_HOST_SYSTEM_NAME=${stdenv.buildPlatform.uname.system}"
           ++ lib.optional (stdenv.buildPlatform.uname.processor != null) "-DCMAKE_HOST_SYSTEM_PROCESSOR=${stdenv.buildPlatform.uname.processor}"
           ++ lib.optional (stdenv.buildPlatform.uname.release != null) "-DCMAKE_HOST_SYSTEM_VERSION=${stdenv.buildPlatform.uname.release}";
+
+          mesonFlags = if mesonFlags == null then null else let
+            # See https://mesonbuild.com/Reference-tables.html#cpu-families
+            cpuFamilies = {
+              aarch64  = "aarch64";
+              armv5tel = "arm";
+              armv6l   = "arm";
+              armv7l   = "arm";
+              i686     = "x86";
+              x86_64   = "x86_64";
+            };
+            crossFile = builtins.toFile "cross-file.conf" (''
+              [properties]
+              needs_exe_wrapper = true
```

Comment:
I don’t think we should set this.

Reply:
Well, it was what was there so I rather change separately, but I do agree. Didn't you make a thing just for this sort of thing?